packages/es-components/src/components/controls/buttons/IconButton.md

```diff
@@ -0,0 +1,50 @@
+Icon buttons will display an icon in a circle with optional text being placed below.
+
+```
+import IconButton from './IconButton';
+
+function IconButtonExample() {
+    const [isHighlighted, setIsHighlighted] = React.useState(false);
+
+    return (
+        <IconButton iconName="man" isHighlighted={isHighlighted} onClick={() => setIsHighlighted(!isHighlighted)}>Text</IconButton>
+    );
+}
```

Comment:
Do you think it would be helpful to add an example with multiple IconButtons together to show the valueof the isHighlighted behavior?

Reply:
It's how we're going to be using it initially, so it probably makes sense to show that off! I'll go ahead and add that.